### PR TITLE
Update command format

### LINKED ISSUE #156: Update UG Command Format
- table for add and edit subcategories
- add tips 

Table can look something like this: 
-------------------------------------------
Add staff | Format: 
                 | Example:

### PR DESCRIPTION
**Pull request by jannaleong**
Writing each command
individually for the add and
edit command is messy.

Write commands in a table
instead.

Fixes #156

### CODE REVIEW

**Review by jamessinmaojun — APPROVED** [review]
lgtm

File: docs/UserGuide.md [hunk 1]
```diff
@@ -106,23 +106,12 @@ Here's a quick look at the different parts of our GUI and some tips on how to us
...
+| Adds a ...        | Format & Examples                                                                                                                                                                                                                                                                                                                  |
+|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Person**        | Format : `/pooch-add ; name : [name] ; phone : [phone] ; address : [address] ; email : [email]` <br> Example: `/pooch-add ; name : Janna  ; phone : 98765435 ; address : Poochie Street 24 ; email : ihelppooches@gmail.com`                                                                                                       |
```
Comment (yleeyilin) [inline docs/UserGuide.md:111]: Maybe can keep the explanation for the examples. Shouldn't the commands be /add-person, /add-staff etc.

**Review by yleeyilin — COMMENTED** [review]
Some minor comments

**Review by yleeyilin — APPROVED** [review]
lgtm

**Pull request merged**